py_ppo.py

```diff
@@ -0,0 +1,191 @@
+"""Download data from the Plant Phenology Data Portal.
+
+This script contains functions to download data from the plant phenology portal
+REST API. This portal has three endpoints:
+
+* https://biscicol.org/api/v1/query - An old version (?) returns json objects
+* https://biscicol.org/api/v2/ppo - Lookup traits from ontology
+* https://biscicol.org/api/v3/download - New version (?) download as zipped csv
+
+The v1 endpoint is not used here because it is a bit inconsistent (e.g. size
+instead of limit, planStructurePrensenceTypes instead of termID, etc.). That
+might only cause confusion.
+
+Useful resources:
+
+* Paper describing the PPO: https://doi.org/10.3389/fpls.2018.00517
+* Data portal frontend: http://plantphenology.org/
+* Data portal backend: https://github.com/biocodellc/biscicol-server
+* Similar R-package: https://docs.ropensci.org/rppo/reference/rppo-package.html
+
+TODO:
+- document options (see https://github.com/ropensci/rppo/blob/master/R/ppo_data.R#L6-L29)
+- write more tests
+- accept multiple inputs for options
+- accept either a single year or a range
+- pythonic formatting of ranges
+- save and reload already downloaded data  #7
+"""
```

Comment:
My previous comment is on the todo list :) Here, I would add accepting a single coordinate with a buffer size too.